### ISSUE TITLE
Document not opening in NC Text after opening sidebar via sharing icon in file list

### ISSUE THREAD

**Issue by violoncelloCH** [nextcloud/text#203]
**To Reproduce**
1. in files view: click on the sharing icon on a Nextcloud Text document so the sidebar opens
2. click on the document to open it in Nextcloud Text
3. Text opens, but it only shows an endless loading spinner
4. See error in js console

**Expected behavior**
document should be opened in Nextcloud Text

**Client details:**
 - OS: Ubuntu 19.04
 - Browser: Firefox
 - Version: 68

<details>
<summary>Server details</summary>
**Text app version:** 1.0.1

**Nextcloud version:** 16.0.3

</details>

<details>
<summary>Logs</summary>

#### Browser log
```
TypeError: "Object(...) is not a function"
    value https://nc.violoncello.ch/apps/text/js/2.ac15b612f4d43ab42353.js:151
    value https://nc.violoncello.ch/apps/text/js/2.ac15b612f4d43ab42353.js:151
    initSession https://nc.violoncello.ch/apps/text/js/2.ac15b612f4d43ab42353.js:172
    mounted https://nc.violoncello.ch/apps/text/js/2.ac15b612f4d43ab42353.js:172
    Ut https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    en https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    insert https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    E https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    hr https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    _update https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    i https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    get https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    run https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    dn https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    ee https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    Xt https://nc.violoncello.ch/apps/viewer/js/viewer.js?v=3b89b802-37:7
    o https://nc.violoncello.ch/core/js/dist/main.js?v=3b89b802-37:230
    H https://nc.violoncello.ch/core/js/dist/main.js?v=3b89b802-37:230
    u https://nc.violoncello.ch/core/js/dist/main.js?v=3b89b802-37:235
viewer.js:7:11601

```

</details>

**Comment by juliushaertl** [comment]
Hm, I cannot reproduce that. Can you try to provide some more detailed steps to reproduce? Do you close the document inbetween?

**Comment by violoncelloCH** [comment]
hmm maybe I could have written it in a clearer way:
opening the sidebar via the share icon needs to happen in the files app
(updated the steps above as well^)
it doesn't happen if in files view you choose three dots > details and go to the sharing tab there, only if you choose the sharing icon in the file list directly

**Comment by juliushaertl** [comment]
I still cannot reproduce this unfortunately. Could you check to expose .map files though your webserver and try to grab an error trace with the actual code line numbers?

**Comment by juliushaertl** [comment]
Ah this might be the same issue that the files_sharing webpack bundle conflicts with the one from the text app, it might be that due to the chunk naming we already have on master, this doesn't occur but to be sure we should check once is in #227

**Comment by juliushaertl** [comment]
Could you recheck if that is fixed for you with 1.0.2?

**Comment by juliushaertl** [comment]
Lets close this as it should be fixed. Feel free to reopen if it occurs again.

**Comment by violoncelloCH** [comment]
sorry for not responding here; I've been quite busy the last days...
just updated the app and checked the issue... can't reproduce it any more so it looks like it's fixed with #227 :) thanks!
